functions.php

```diff
@@ -56,4 +56,22 @@ function displayPokemon(array $pokemons): string {
         }
     }
     return $result;
+}
+
+
+function addPokemonToDatabase() {
+
```

Comment:
Probably best to remove all the blank lines in between your code, for PSR2

Reply:
Removed.